code/modules/projectiles/guns/projectile/shotgun/gladstone.dm

```diff
@@ -0,0 +1,74 @@
+/obj/item/gun/projectile/shotgun/pump/gladstone
+	name = "\"Gladstone\" combat shotgun"
+	desc = "A venerable shotgun with incredibly dubious patent claims by H&S Arms, it serves as a common shotgun alongside a number of virtually identical weapons across the galaxy. \
+	Can hold up to 7+1 20mm shells in its tube magazine."
+	icon = 'icons/obj/guns/projectile/gladstone.dmi'
+	icon_state = "gladstone"
+	item_state = "gladstone"
+	origin_tech = list(TECH_COMBAT = 4, TECH_MATERIAL = 2)
+	max_shells = 7
+	caliber = CAL_SHOTGUN
+	damage_multiplier = 1.1
+	penetration_multiplier = 1.1 //Combat shotgun shouldn't be worse than makeshift shotgun
+	ammo_type = /obj/item/ammo_casing/shotgun
+	fire_sound = 'sound/weapons/guns/fire/shotgun.ogg'
+	matter = list(MATERIAL_PLASTEEL = 20, MATERIAL_PLASTIC = 6)
+	price_tag = 850
+	init_recoil = RIFLE_RECOIL(1.2)
+	saw_off = TRUE
+	sawn = /obj/item/gun/projectile/shotgun/pump/gladstone/sawn
+	serial_type = "H&S"
+
+	wield_delay = 1 SECOND
+	wield_delay_factor = 0.4 // 40 vig
+	gun_parts = list(/obj/item/part/gun/frame/gladstone = 1, /obj/item/part/gun/grip/black = 1, /obj/item/part/gun/mechanism/shotgun = 1, /obj/item/part/gun/barrel/shotgun = 1)
+
+/obj/item/part/gun/frame/gladstone
+	name = "Gladstone frame"
+	desc = "A Gladstone shotgun frame. The everyman's shotgun."
+	icon_state = "frame_gladstone"
+	result = /obj/item/gun/projectile/shotgun/pump/gladstone
+	resultvars = list(/obj/item/gun/projectile/shotgun/pump/gladstone)
+	gripvars = list(/obj/item/part/gun/grip/black)
+	mechanismvar = /obj/item/part/gun/mechanism/shotgun
+	barrelvars = list(/obj/item/part/gun/barrel/shotgun)
+
+/obj/item/gun/projectile/shotgun/pump/gladstone/sawn
+	name = "\"Gladstone\" stakeout shotgun"
+	desc = "A venerable shotgun that's been destroyed by some sort of heartless monster. Can hold up to 4+1 20mm shells in its tube magazine."
+	icon = 'icons/obj/guns/projectile/sawnoff/gladstone.dmi'
+	icon_state = "obrez"
+	item_state = "obrez"
+	gun_parts = list(/obj/item/part/gun/frame/gladstone = 1, /obj/item/stack/material/plastic = 1, /obj/item/part/gun/mechanism/shotgun = 1, /obj/item/stack/material/plasteel = 2)
```

Comment:
sawns shouldnt give back the frame iirc

Reply:
Was based on regulator/grizzly which both give frames when sawed off

Reply:
Odd they really shouldnt, so you have a reason to use the repair repices

Reply:
Repairs.dm is commented out, you can't repair guns anymore

Reply:
Fair

Reply:
IMHO this is good. Frame + mechanism, the only tings that don't get fully manged(i guess we'll assume that stock=handle and barrel=barrel.)